charts/vsphere-csi/v2.1.0/templates/vsphere-csi-controller-deployment.yaml

```diff
@@ -44,9 +44,9 @@ spec:
           volumeMounts:
             - mountPath: /csi
               name: socket-dir
-{{- if .Values.csiResizer.enabled }}
+{{- if .Values.csiController.csiResizer.enabled }}
         - name: csi-resizer
-          image: quay.io/k8scsi/csi-resizer:v1.0.0
+          image: "{{ template "system_default_registry" . }}{{ .Values.csiController.image.csiResizer.repository }}:{{ .Values.image.csiController.csiResizer.tag }}"
```

Comment:
The tag is swapped.

It should be 

.Values.csiController.image.csiResizer.tag }}"

Reply:
fixed